### DISCUSSION TITLE
The diesel support

### DISCUSSION THREAD

**Discussion by fernandobatels** [fernandobatels/rsfbclient#128] [General]
The diesel implementation is ready. Now I'm waiting a new [diesel version](https://github.com/diesel-rs/diesel/releases) to publish the crate.

```
Projetos/rsfbclient/rsfbclient-diesel ‹master› % cargo publish
    Updating crates.io index
error: all dependencies must have a version specified when publishing.
```

**Comment by weiznich** [comment]
I've published diesel 2.0.0 yesterday. You should be fine to go now.

**Reply by fernandobatels** [reply]
Released: https://crates.io/crates/rsfbclient-diesel

Thank you for your help!!
